data/mods/MindOverMatter/modinfo.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "type": "MOD_INFO",
+    "id": "mindovermatter",
+    "name": "Mind Over Matter",
+    "authors": [ "StandingStorm" ],
+    "maintainers": [ "StandingStorm" ],
+    "description": "Psionic powers for Cataclysm",
+    "category": "content",
+    "dependencies": [ "dda" ]
+  }
+]

```

Comment:
```suggestion
]

```

Reply:
Can you also move this file to the same level as the readme?

Reply:
I'm afraid I'm not sure what you mean--aren't they already at the same level? They're both in the base MindOverMatter directory.